.travis.yml

```diff
@@ -0,0 +1,34 @@
+---
+language: minimal
+
+services:
+  - docker
+
+jobs:
+  include:
+    - stage: Build and Tests
+      name: Build
+      language: go
+      go: 1.14.x
+      script: make build
+    - name: Unit tests
+      language: go
+      go: 1.14.x
+      script: make unit-test
+    - name: Markdown spellcheck
+      script: make readme-test
+    - name: License test
+      if: type = push AND env(GITHUB_TOKEN) IS present
+      language: go
+      go: 1.14.x
+      script: make license-test
+    - name: Go report card test
+      script: make go-report-card-test
+    - name: AEIQ E2E tests
```

Comment:
This needs a `if: type = push AND env(GITHUB_TOKEN) IS present` check because AWS env vars are not available/exposed for PRs